source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -270,7 +270,12 @@ export default class CreepVote extends React.Component {
         <div className="mb-5">
           <div className="col-12 text-center">
             <h3 className="h2-heading mb-1">
-              {this.state.totalVotes + 1} Votes — invite your friends!
+              <Localized
+                id="voted-header"
+                $voteCount={this.state.totalVotes + 1}
```

Comment:
This looks like the kind of non-standard magic I'd love to keep out of a codebase. Is there a way to achieve the same thing without relying on a completely non-standard method and syntax for variable declarations?

Reply:
This is a Fluent-ism: props to `<Localized>` which start with `$` are passed into translations as arguments.